submissions/misshatsker/document-object-model/index.js

```diff
@@ -0,0 +1,122 @@
+        propertyLifespan: "Up to 20 years",
+        propertyDiet: "Balanced diet",
+        propertySize: "1/2 to 2 inches",
+        propertyWeight: "About 8 ounces"
+    },
+    {
+        title: "Ants",
+        media: "./media/ant.jpg",
+        description: "Ants first rose during the Cretaceous period around 130 million years ago! They have survived the Cretaceous-Tertiary (K/T extinction) that killed the dinosaurs as well as the ice age. They have the ability to carry between 10 and 50 times their own body weight! The amount an ant can carry depends on the species. This amazing strength is a result of their small size. ",
+        propertyLifespan: "Up to 30 years",
+        propertyDiet: "Omnivore",
+        propertySize: "0.030 to 2.0 inches",
+        propertyWeight: "1 to 5 mg"
+    }
+];
+
+const menuIcon  = document.getElementsByClassName("menu-icon")[0];
+const menu = document.getElementsByClassName("menu")[0];
+
+const media = document.getElementsByClassName("media-animal")[0];
+
+const documentMainInfo = document.getElementsByClassName("animal-main-info")[0];
+
+const title = documentMainInfo.getElementsByClassName("animal-category-title")[0];
+const description = documentMainInfo.getElementsByClassName("animal-category-description")[0];
+const propertyLifespan = documentMainInfo.getElementsByClassName("property-description-lifespan")[0];
+const propertyDiet = documentMainInfo.getElementsByClassName("property-description-diet")[0];
+const propertySize = documentMainInfo.getElementsByClassName("property-description-size")[0];
+const propertyWeight = documentMainInfo.getElementsByClassName("property-description-weight")[0];
+
+
+function changeMainPageInfo(animal) {
+    title.textContent = animal.title;
+    media.src = animal.media;
+    description.textContent = animal.description;
+    propertyLifespan.textContent = animal.propertyLifespan;
+    propertyDiet.textContent = animal.propertyDiet;
+    propertySize.textContent = animal.propertySize;
+    propertyWeight.textContent = animal.propertyWeight;
+}
+
+changeMainPageInfo(animals[0]);
+
+let activeLi = null;
+for (let i = 0; i < animals.length; i++) {
+    const animal = animals[i];
+    const animalTitle = animal.title;
+
+    let li = document.createElement('li');
+    li.className = "menu-item";
+    if (i === 0) {
+        li.classList.add("active-menu-item");
+        activeLi = li;
+    }
+
+    li.innerHTML = '<a class="link" href="#'+ animalTitle + '">' + animalTitle + '</a>';
+
+    li.onclick = () => {
+        closeMenu();
+
+        if (activeLi) {
+            activeLi.classList.remove("active-menu-item");
+        }
+
+        changeMainPageInfo(animal);
+        li.classList.add("active-menu-item");
+        activeLi = li;
+    };
+
+    menu.append(li);
+}
+
+let isOpened = false;
+menuIcon.onclick = () => {
+    if (isOpened) {
```

Comment:
You can skip additional var `isOpened` if you will use `menuIcon.classList.contains`